storage/post_policy_v4.go

```diff
@@ -52,22 +52,31 @@ type PostPolicyV4Options struct {
 	// Exactly one of PrivateKey or SignBytes must be non-nil.
 	PrivateKey []byte
 
-	// SignBytes is a function for implementing custom signing. For example, if
+	// SignBytes is a function for implementing custom signing.
```

Comment:
Maybe include a sentence about how it differs from `SignRawBytes` even though we are deprecating it.